spec/spec_helper.rb

```diff
@@ -13,6 +13,8 @@
 # Load all support files
 Dir["#{File.dirname(__FILE__)}/support/**/*.rb"].each { |f| require f }
 
+$TESTING = true
+
```

Comment:
Is this required or did you forget it here?

Reply:
It is required because ironically using POST makes tests fail because CSRF protection kicks in.
This is how sidekiq deals with the issue (https://github.com/mperham/sidekiq/blob/master/lib/sidekiq/web.rb#L154)